src/library.js

```diff
@@ -4836,6 +4836,7 @@ LibraryManager.library = {
   emscripten_asm_const_double_sync_on_main_thread: function() {},
   emscripten_asm_const_async_on_main_thread: function() {},
 
+#if !WASM_BACKEND
   // ======== compiled code from system/lib/compiler-rt , see readme therein
```

Comment:
Unrelated?

Reply:
No, this is needed or Closure will complain that the generated functions below have unnecessary arithmetic operations.